packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -163,73 +161,125 @@ This is prototyped in the [formatted-text-column branch](https://github.com/ni/n
 
 Nimble will provide several columns that derive from the above base classes and provide higher level text formatting APIs for specific data types.
 
+All columns will include a `field-name` string attribute to select the field to display and a `placeholder` string attribute to specify the text to display if the field value isn't in the expected format (more details about expected formats for each column type).
+
+All columns will offer standard Angular and Blazor integration.
+
 #### Column naming
 
-Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs
-/table-columns-hld.md#column-type-philosophy) where:
+Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs/table-columns-hld.md#column-type-philosophy) where:
 
 -   category: `table-column`
 -   presentation: `text`
--   variants: Different variants are allowed for configurations that vary significantly / don't make sense to add to `table-column-text`, ie for a `table-column-numeric-text` or `table-column-date-text`
+-   variants: Different variants are allowed for configurations that vary significantly / don't make sense to add to `table-column-text`, ie for a `table-column-number-text` or `table-column-date-text`
 
 #### Text column
 
 `nimble-table-column-text` will continue to present the same API it does today, but will derive from the base classes described above.
 
-#### Numeric column
+#### Number column
+
+Nimble will introduce `nimble-table-column-number-text` which formats a numeric field value and displays it as text. It will offer sufficient configuration to support use cases 1-4 above.
+
+##### API
 
-Nimble could introduce `nimble-table-column-numeric-text` which formats a numeric field value and displays it as text. It will offer sufficient configuration to support use cases 1-4 above.
+-   `prefix` - a string which will be appended before each value (e.g. `'$'`). Defaults to `''`. Should only be used if formatting can't be achieved via `format`.
+-   `suffix` - a string which will be appended after each value (e.g. `'%'` or `' V'`). Defaults to `''`. Should only be used if formatting can't be achieved via `format`. Spacing will be at the discretion of clients, but Nimble will recommend including a space before the unit except for symbol units like `%`, `"`, `°`, and `°C`. This [matches the Chicago Manual of Style](https://www.chicagomanualofstyle.org/book/ed17/part2/ch09/psec016.html) (requires VPN) which is NI's documentation style guide.
+-   `alignment` - a string value matching `"left"`, `"right"`, or `undefined` (the default, meaning `"automatic"`) which controls whether values and column headers are left or right aligned within the column. If set to `undefined` Nimble will choose left or right based on the value of `format`. Clients should select `right` if it is known that the decimal separators of all values in the column will align in the given the `format`.
```

Comment:
Should we use `start` and `end` as more ltr/rtl friendly terms?

Reply:
From my reading, both `start`/`end` and `prefix`/`suffix` terminology imply being based of the flow of the language. i.e. for words in RTL, both `start` and `prefix` would mean "before the text" which is "on the right".

However, based on [the rtl unit formatting post on StackExchange](https://linguistics.stackexchange.com/questions/16865/order-of-components-within-measurement-units-in-rtl-languages) that you linked in another thread, numbers and units are typically formatted LTR even in RTL languages. So if we wanted to be more explicit that the prefix will always be on the left, we could go with terminology like `left-text` and `right-text`.

But I think it'd be clearer to stick with the `prefix` and `suffix` terminology and just document that our current proposal is that text is always formatted RTL so `prefix` always means `left`. I'll add a note to the spec.

(I also considered switching to `start`/`end` for a different reason: to align with web APIs like [`align-items: flex-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). Or `before`/`after` to align with pseudoelements. But `prefix`/`suffix` seems really common in typography so I'd like to stick with it.)